src/Snippets/Docs/HttpClientIntegrations.cs

```diff
@@ -0,0 +1,129 @@
+using System.Net;
+using System.Net.Http;
+using Flurl.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Polly.Retry;
+using Refit;
+using RestSharp;
+
+namespace Snippets.Docs;
+
+internal static class HttpClientIntegrations
+{
+    private static readonly Uri DownstreamUri = new("https://httpstat.us/408");
```

Comment:
Let's call this `BaseAddress`. Also remove the path, as a path is being specified in all the examples anyway.